serving/src/test/java/feast/serving/it/ServingServiceBigTableIT.java

```diff
@@ -199,6 +232,8 @@ static void globalSetup() throws IOException {
         7200);
 
     // BigTable Table names
+    String superLongBtTableName = String.format("%s__%s", projectName, superLongEntityName);
+    superLongBtTableName = trimAndHash(superLongBtTableName, 50);
```

Comment:
This test doesn't make sense, bc it compares function (under test) which may contain bugs with itself. It's even better to hardcode here expected value (of table name).